readme.md

```diff
@@ -253,6 +253,7 @@
 - [Machine Learning](https://github.com/josephmisiti/awesome-machine-learning)
 	- [Tutorials](https://github.com/ujjwalkarn/Machine-Learning-Tutorials)
 	- [ML with Ruby](https://github.com/arbox/machine-learning-with-ruby) - Learning, implementing, and applying Machine Learning using Ruby.
+	- [Core ML Models](https://github.com/likedan/Awesome-CoreML-Models) - List of machine learning models in Core ML format.
```

Comment:
This should be a description of `Core ML Models`, not the list itself.

For example:

> Models for Apple's machine learning framework.